app/src/main/java/com/framgia/forder/screen/createshop/CreateshopFragment.java

```diff
@@ -21,11 +25,17 @@
 import com.framgia.forder.databinding.FragmentCreateShopBinding;
 import com.framgia.forder.utils.navigator.Navigator;
 import com.framgia.forder.widgets.dialog.DialogManager;
+import java.io.ByteArrayOutputStream;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
```

Comment:
ERROR: Unused import: java.io.InputStream. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports